docs/products/flink/howto/connect-bigquery.rst

```diff
@@ -0,0 +1,201 @@
+
+::
+
+    avn service integration-create 
+        --source-endpoint-id <source-endpoint-id>
+        --dest-service <flink-service-name>
+        -t flink_external_bigquery
+
+For example,
+
+::
+
+    avn service integration-create 
+        --source-endpoint-id eb870a84-b91c-4fd7-bbbc-3ede5fafb9a2 
+        --dest-service flink-1 
+        -t flink_external_bigquery
+
+
+where:
+
+* ``--source-endpoint-id``: The ID of the integration endpoint you want to use as the source. In this case, it is the ID of the external Google BigQuery integration endpoint. In this example, the ID is ``eb870a84-b91c-4fd7-bbbc-3ede5fafb9a2``.
+* ``--dest-service``: The name of the Aiven for Apache Flink service you want to integrate with the external BigQuery endpoint. In this example, the service name is ``flink-1``.
+* ``-t``: The type of integration you want to create. In this case, the ``flink_external_bigquery`` integration type is used to integrate Aiven for Apache Flink with an external BigQuery endpoint.
+
+Step 5: Verify integration with service
+`````````````````````````````````````````
+After creating the integration between Aiven for Apache Flink and and Google BigQuery, the next step is to verify that the integration has been created successfully and create Aiven for Apache Flink applications that use the integration.
+
+To verify that the integration has been created successfully, run the following command:
+
+::
+
+    avn service integration-list --project <project-name> <flink-service-name>
+
+For example:
+
+::
+
+    avn service integration-list --project systest-project flink-1
+
+
+where:
+
+* ``--project``: The name of the Aiven project that contains the Aiven service you want to list integrations for. In this example, the project name is ``systest-project``.
+* ``flink-1``: The name of the Aiven service you want to list integrations for. In this example, the service name is ``flink-1``, which is an Aiven for Apache Flink service.
+
+To create Aiven for Apache Flink applications, you will need the integration ID of the Aiven for Apache Flink service. Obtain the ``integration_id`` from the integration list.
+
+Step 6: Create Aiven for Apache Flink applications
+`````````````````````````````````````````````````````
+
+With the ``integration ID`` obtained from the previous step, you can now create an application that uses the integration. For information on how to create Aiven for Apache Flink applications, see :ref:`avn service flink create-application <avn service flink create-application>`.
+
+Following is an  example of a Google BigQuery SINK table: 
+
+.. code:: sql 
+
+    CREATE TABLE `table1` (
+        `name` STRING
+    )
+    WITH
+    (
+        'connector' = 'bigquery',
+        'Service-account' = '',
+        'project-id'= '',
+        'dataset' = 'bqdataset',
+        'table' = 'bqtable',
+        'Table-create-if-not-exists' = 'true',
```

Comment:
lower case.  `table-create-if-not-exists`

Reply:
Fixed this.